docker/opencv/Dockerfile

```diff
@@ -0,0 +1,15 @@
+
+FROM demisto/python3-deb:3.9.1.14969
+
+COPY requirements.txt .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+&& apt install python3 python3-distutils wget libglib2.0-0 libsm6 libxext6 libgl1-mesa-glx libxrender-dev -y \
```

Comment:
You should install also libxrender (not only the `-dev`). and in the purge section remove the `libxrender-dev` install.

Aslo I am not clear why you are installing `python3 python3-distutils` . I don't think this should be needed.

Reply:
done